<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QWidget" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>671</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>671</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/new/icon/image/icon.jpg</normaloff>:/new/icon/image/icon.jpg</iconset>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>672</width>
     <height>479</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="welcom_icon">
      <property name="minimumSize">
       <size>
        <width>670</width>
        <height>250</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>670</width>
        <height>250</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/welcome/image/welcome_icon.jpg);</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>670</width>
        <height>220</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>670</width>
        <height>220</height>
       </size>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>81</x>
         <y>21</y>
         <width>522</width>
         <height>194</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="person_icon">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>100</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>100</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/new/person/image/person_icon.jpg);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLineEdit" name="qq_number">
                <property name="minimumSize">
                 <size>
                  <width>250</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>250</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="placeholderText">
                 <string>请输入账号</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="register_2">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>注册账号</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLineEdit" name="password">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>250</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>250</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="placeholderText">
                 <string>请输入密码</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="find_password">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>找回密码</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QRadioButton" name="rem_password">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>记住密码</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="auto_login">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>自动登录</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="logi_button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(85, 85, 255);</string>
            </property>
            <property name="text">
             <string>立即登录</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="tip_label">
            <property name="minimumSize">
             <size>
              <width>350</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>350</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
